### DISCUSSION TITLE
Check if minGasReserve was changed.

### DISCUSSION THREAD

**Discussion by kraikov** [PaulRBerg/prb-proxy#27] [Q&A] [answered]
There is a check whether the `owner` was changed after the `delegatecall`, but not for the `minGasReserve`. Why is that?

**Comment by PaulRBerg** [comment, marked as answer]
Hey @kraikov, good question!

PRBProxy prevents the `owner` from being modified due to security reasons. If the owner really wants to transfer ownership, they can call the [transferOwnership](https://github.com/paulrberg/prb-proxy/blob/1d6c1586890611e6c5dac1d8c10bbc0e300ab7b5/contracts/PRBProxy.sol#L139-L145) function.

There is no check for `minGasReserve` because that variable is not involved in access control. If a malicious scripts overrides the gas reserve, the owner can just reset it back to whatever value they want via the [setMinGasReserve](https://github.com/paulrberg/prb-proxy/blob/1d6c1586890611e6c5dac1d8c10bbc0e300ab7b5/contracts/PRBProxy.sol#L118-L123) function.

**Reply by kraikov** [reply]
Isn't that a good attack vector? If I want to harm someone I can front-run them each time they use the proxy and set the `minGasReserve` to 0 which would lead to always failing transactions. It would cost me a lot, but it's doable. WDYT?

**Reply by PaulRBerg** [reply]
You can't front-run them because the `execute` function can only be called by the owner. It's only the owner who, accidentally, can set the `minGasReserve` to 0. But then (s)he can always set it back to an appropriate value.

**Reply by kraikov** [reply]
Oh yeah, sorry. I was modifying the proxy to work for my needs. And in my case, the proxy can be called by anyone, that's why I got confused. 
Thanks!
